docs/2.9. Usage - Fully Automated Testing of Controllers.md

```diff
@@ -0,0 +1,179 @@
+# Testing Controllers
+
+## Fully Automated Controller Tests
+
+The testing tool supports fully automated testing of Controllers by replacing the Testing Façade with your own application to receive and process the requests from the NMOS Testing Tool. A full example utilising Flask and Selenium can be found on the nmos-js repository under AutomatedTesting. 
+
+## Building your fully automated test interface
+
+The Testing Façade IP/Hostname and Port on the test suite should be set to the location of your application.
+POST requests are sent from the Test Suite to the API endpoint '{IP/hostname}:{Port}/x-nmos/testingfacade' 
+
+The JSON schema for the Q&A protocol is as follows:
```

Comment:
Assume this will be updated with links to APIs and examples, like the _docs/5.1. Extension - Adding New Tests.md_ docs?